cmd/root.go

```diff
@@ -76,6 +76,8 @@ func Execute() {
 		fmt.Println(err)
```

Comment:
Do we not need to also flush the log here in case of an error?

Reply:
yes, I agree, Added `glog.flush() before exit on error`.